components/Footer.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Image from 'next/image';
+import { FooterLogo, Instagram, Facebook, Linkin, Twitter, Call, Mail, Location } from '../assets';
+
+const Footer = () => {
+  return (
+    <footer className="bg-amcovad-tertiary bg-cover bg-left lg:bg-top  bg-[url('../assets/images/footer-hexagons.png')]">
+      <div className="px-4 pt-16 mx-auto sm:max-w-xl md:max-w-full pb-24 lg:max-w-screen-xl md:px-24 ">
+        <div className="container px-2 py-4 mx-auto">
+          <div className="grid gap-4 lg:gap-20 row-gap-10 mb-8 lg:grid-cols-6">
+            <div className="lg:col-span-2">
+              <a href="#" aria-label="Go home" title="Company" className="inline-flex items-center">
+                <Image src={FooterLogo} alt="footer logo" />
+              </a>
+              <div className="mt-4 lg:max-w-sm">
+                <div className="flex flex-col justify-between pt-1 pb-10 sm:flex-row">
+                  <div className="flex items-center mt-4 space-x-4 sm:mt-0">
+                    <a
```

Comment:
Can we use the next Link directly instead of using the `a` tag. 

Even if we don't have the right URL, we can use https://instagram.com, https://facebook.com and https://twitter.com.

Also, we can create a content file to save this too to make it easier to update in the future.